deployment/terraform/container_service.tf

```diff
@@ -25,6 +25,26 @@ resource "aws_security_group" "app" {
   }
 }
 
+resource "aws_security_group" "container_instance" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgContainerInstance"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_security_group_rule" "container_instance" {
```

Comment:
Good catch here. Come to think of it, is there any reason why the Batch container instances can't use the security group configured for the Fargate container instances? That seems to already have all the right ingress/egress rules for the bastion and database.

Reply:
They could. We'd want to rename the app security group though to remove `EcsService`. I'm leaning against this though, because the app security group contains rules for the load balancer we wouldn't want applied to the Batch container instances.

Reply:
Gotcha. I think some duplication makes sense here in order to keep the two security groups separate.